src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(RandomColor.values().length);
```

Comment:
```suggestion
public class ColorSupplier {
    private static final Random ranodm = new Random();
    
    public String getRandomColor() {
        int index = random.nextInt(RandomColor.values().length);
```
lets make this object global, this way Random will be created just once, not every time this method is invoked